.github/ISSUE_TEMPLATE/use-case-suggestion.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Use cases suggestion
+about: Suggestion for improving use cases document
+---
+
+## On use case
+
+Which existing use case is your suggestion intended for? (or a new use case)
+
+## Details
+
+
+
+## Acceptance criteria
+
+What actions are needed to resolve this issue? (checklist)
+
+* [ ] 
```

Comment:
Strange that this didn't include the following as given, I wonder why:

[ ] Proposer will i) implement ii) share implementation publicly -- See also uc-survey.

Reply:
because strangely you are commenting on closed PR rather than creating new PR proposing that change 😉